cabal.project

```diff
@@ -120,7 +120,7 @@ source-repository-package
   location: https://github.com/input-output-hk/iohk-monitoring-framework
   tag: d8f35c3c725be67148681227f9d64876f5909120
   --sha256: 09v1m9km3arzxaqlyaw576w69fwfi02q7nm5ias162sfzzg8za4x
-  subdir:   contra-tracer
+  subdir: contra-tracer
 
 source-repository-package
```

Comment:
Could you also transform `cardano-prelude`?

Reply:
No, see the description of the second commit:

> For some reason, using `.` as a subdir doesn't seem to be supported. For now,
> work around it by having two entries for `cardano-prelude`.